main.py

```diff
@@ -162,6 +162,16 @@ def handle_file(update, context):
 def handle_button_click(update, context):
     return handle('button_click', update, context)
 
+async def send_message(bot, chat_id, text, reply_to_message_id=None, reply_markup=None):
+    max_length = 4096
+    for i in range(0, len(text), max_length):
+        await bot.send_message(
+            chat_id=chat_id,
+            text=text[i:i+max_length],
+            reply_to_message_id=reply_to_message_id if i == 0 else None,
+            reply_markup=reply_markup if i == 0 else None
+        )
```

Comment:
How do you handle the summary for multiple message? I didn't see the implementation.

In this case, the response won't be accurate since the context are separated.